irods/test/resource_test.py

```diff
@@ -0,0 +1,40 @@
+#! /usr/bin/env python
+from __future__ import absolute_import
+import os
+import six
+import sys
+import unittest
+
+from irods.test import helpers
+
+class TestResource(unittest.TestCase):
+
+    from helpers import create_simple_resc_hierarchy
+
+    def setUp(self):
+        self.sess = helpers.make_session()
+
+    def test_new_resource_properties_for_parent_and_hierarchy__392(self):
+        ses = self.sess
+        root = 'pt_392'
+        leaf = 'leaf_392'
+        with self.create_simple_resc_hierarchy(root,leaf) as resc_hier:
```

Comment:
please make the test at least three levels... rather than just parent/child.

Reply:
Oh... like deferred/pt/leaf

Reply:
sure, or even just pt/pt/leaf - mostly that we squeeze out any corner cases and assumptions.

Reply:
Ok, done... I think.